src/app/home/avatar-generator/selector/atribute-selector.html

```diff
@@ -0,0 +1,32 @@
+<!-- Rounded dialog -->
+<section>
+  <div class="card nes-container with-title" (click)="openModal()">
+    <p class="title">{{decorator.name}}</p>
+    <div [innerHTML]="getIcon() | sanitizeHtml"></div>
+  </div>
+
+  <dialog class="container nes-dialog is-rounded" [id]="dialogId">
+    <button type="button" class="nes-btn is-error close-btn pixel-text white-text" (click)="closeModal()">X</button>
+    <form method="dialog">
+      <div class="row">
+        <p class="pixel-title">Selecciona un tipo {{decorator.name}}</p>
+        <div class="col s12 m6">
+          <div [innerHTML]="getAvatar() | sanitizeHtml"></div> 
```

Comment:
This method and the other methods that are also called in this template are getting called a lot of times even when there are no changes in the avatar. It would be good to optimize this to avoid calling them many times. Two possible approaches could be:
1. Making this component use the OnPush change detection strategy.
2. Building these svgs just when it's necessary and storing the results in variables, then the template can reference these variables instead of calling the methods.